proto/v2/observation.proto

```diff
@@ -62,17 +62,13 @@ message FacetObservation {
 }
 
 message FacetFilter {
-  // When specified, only observations with this unit are returned.
-  string unit = 1;
-  // When specified, only observations with this measurement method are
-  // returned.
-  string measurement_method = 2;
-  // When specified, only observations with this observation period are
-  // returned.
-  string period = 3;
+  // List of variables that this facet filter applies to
+  repeated string variables = 1;
   // When specified, only observations with provenance in these domain are
```

Comment:
nit: this domain is... also can change the proto field to string domain = 2, ... facet_id = 3

Reply:
done